### PR TITLE
Adds Android guide

### PR DESCRIPTION
**Pull request by acadet**
Publishes first version

### CODE REVIEW

File: Android.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+#### Avoid Java hashes
+
+Android development implies to be careful about performances and memory management. Instead of using Java hashes (such as `HashMap`), you should use built-in ones, as `SparseArray`. They are designed for being reverent of Android devices. 
+
+#### Manages your memory
+
+Memory is an important resource and quite limited. You should be attentive to free objets, resources and assets as soon as possible. Stop operations when your activity is hidden. Do not let services running for peccadilloes.
+
+#### Do not use enums
```
Comment (btaitelb) [inline Android.md]: Can we change this to "prefer static constants over enums"? I think there are still some cases where enums are better, like when we want a lightweight class that includes functionality.

File: Android.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+Enums are almost 3x eager in memory than static fields. You should use those ones instead, even if for integers.
+
+#### Use static methods (if possible)
+
+Static calls may be 3x faster than classic ones. Use them as much as possible, especially when you need no fields or context (e.g. helpers).
+
+#### Use `float` instead of `double`
+
+Double uses 2x more space than float. What are you still using them?
```
Comment (btaitelb) [inline Android.md]: I remember hearing this advice back when everyone was using 32-bit processors, but with 64-bit processors, is it really an issue still?

**Comment by btaitelb** [comment]
+1

**Comment by acadet** [comment]
Updated guide

**Comment by btaitelb** [comment]
+1

**Pull request merged**